bigquery/README.md

```diff
@@ -12,21 +12,24 @@ if you haven't done so already.
     bq --project_id MY_GOOGLE_CLOUD_PROJECT mk platinum_genomes
     bq --project_id MY_GOOGLE_CLOUD_PROJECT cp google.com:biggene:platinum_genomes.sample_info  MY_GOOGLE_CLOUD_PROJECT:platinum_genomes.sample_info
     ```
-  * Change project ids in `config/platinum_genomes/facet_fields.csv`.
+  * Change project ids in `dataset_config/platinum_genomes/facet_fields.csv`.
 * If you want to use your own dataset:
-  * Create a `config/private` directory. (Files in `config/private` won't be
-added to this git repo.) Make a new directory under `config/private` and copy
-`config/template/*` to it.
+  * For each dataset, there is a directory in `dataset_config`, e.g.
+  `dataset_config/amp_pd`. Create a directory for your dataset. Copy
+  `dataset_config/template/*` to this directory.
   * Edit config files; instructions are in the files. Read
   [Overview](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#overview)
   for some background information.
 * If `~/.config/gcloud/application_default_credentials.json` doesn't exist, create it by running `gcloud auth application-default login`.
-* If using default dataset: `docker-compose up --build`  
-  If using custom dataset: `DATASET_CONFIG_DIR=config/private/MY_DATASET docker-compose up --build`
+* If using default dataset: `docker-compose up --build`
+  If using custom dataset: `DATASET_CONFIG_DIR=dataset_config/MY_DATASET docker-compose up --build`
 * View Elasticsearch index at
  `http://localhost:9200/platinum_genomes/_search?pretty=true`. If using your
  own dataset, change `platinum_genomes` to your dataset name.
 
+If using your own dataset: Now that your dataset is indexed, follow
+https://github.com/DataBiosphere/data-explorer to bring up a Data explorer UI.
```

Comment:
Link directly to deploy/ directory which contains the relevant README instructions.

Reply:
Reader doesn't necessarily want to deploy, they could just want to bring up local deployment.